examples/visual-tests/amp-story/amp-story-tooltip-desktop.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const {verifyCssElements} = require('../../../build-system/tasks/visual-diff/helpers');
+
+module.exports = {
+  'tapping on a clickable anchor should show the tooltip': async (page, name) => {
+    await page.tap('.next-container > button.i-amphtml-story-button-move');
+    await page.waitFor(1000);
```

Comment:
Visual diff tests don't "really" run in parallel, they _load_ in parallel but are executed sequentially. If you can make this number smaller, it'll keep things running faster. This isn't a big deal on it's own (you're adding about 9 seconds of wait total in these two test cases) but these things add up :)

Reply:
Added a commit that is smarter about when to use `waitFor` using ms.